### PR TITLE
Updated Service Bus Queues binding for components-contrib#1791

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Updated docs to reflect options added in dapr/components-contrib#1791

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md [hunk 1]
```diff
@@ -47,10 +49,14 @@ The above example uses secrets as plain strings. It is recommended to use a secr
...
 | `minConnectionRecoveryInSec` | N | Input | Minimum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. Default: `2` | `5` |
 | `maxConnectionRecoveryInSec` | N | Input | Maximum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. After each attempt, the binding waits a random number of seconds, increasing every time, between the minimum and the maximum. Default: `300` (5 minutes) | `600` |
+| `maxActiveMessages`     | N  |Defines the maximum number of messages to be processing or in the buffer at once. This should be at least as big as the maximum concurrent handlers. Default: `1` | `1`
+| `maxConcurrentHandlers` | N  |Defines the maximum number of concurrent message handlers. Default: `1`. | `1`
+| `lockRenewalInSec`      | N  |Defines the frequency at which buffered message locks will be renewed. Default: `20`. | `20`
+| `timeoutInSec` | N | Input/Output | Timeout for all invocations to the Azure Service Bus endpoint, in seconds. *Note that this option impacts network calls and it's unrelated to the TTL applies to messages*. Default: `60` | `60` |
```
Comment (ItalyPaleAle) [inline daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md:65]: This isn't a new option but looks like it was missing

File: daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md [hunk 2]
```diff
@@ -47,10 +55,14 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `connectionString` | Y | Input/Output | The Service Bus connection string. Required unless using Azure AD authentication. | `"Endpoint=sb://************"` |
 | `namespaceName`| N | Input/Output | Parameter to set the name of the Service Bus namespace. Required if using Azure AD authentication. | `"namespace"` |
 | `queueName` | Y | Input/Output | The Service Bus queue name. Queue names are case-insensitive and will always be forced to lowercase. | `"queuename"` |
-| `ttlInSeconds` | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `"60"` |
+| `ttlInSeconds` | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `86400` |
```
Comment (ItalyPaleAle) [inline daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md:58]: Changed this so it's different from `timeoutInSec` and it helps giving a better idea of what this is

Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md:58]: @ItalyPaleAle  - Can you also change the suggested value in the YAML above. This below
   - name: ttlInSeconds # Optional
     value: 60. <<<

Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md:58]: Looks like you already did it. Thanks!

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**